LICENSE

```diff
@@ -1,28 +1,21 @@
-Copyright (c) 2017, alchemtest developers
```

Comment:
Keep the original license file. I don't want to have to think like a lawyer and figure out if the old and the new one are equivalent. Just keep the old file verbatim.